rio_tiler/utils.py

```diff
@@ -299,6 +302,19 @@ def get_vrt_transform(
         # Get Bounds for the rounded window
         bounds = src_dst.window_bounds(w)
 
+    elif align_bounds_with_dataset:
```

Comment:
I think this code is the same as in reader.py - should it be a function in this file and re-used in reader.py?